src/writer/app_runner.py

```diff
@@ -104,6 +106,7 @@ def __init__(self,
         self.app_path = app_path
         self.mode = mode
         self.run_code = run_code
+        self.source_files: SourceFilesDirectory = {"children": {}, "type": "directory"}
```

Comment:
I think the attribute is not used. You should remote it.

Reply:
done in https://github.com/writer/writer-framework/pull/724/commits/8de3fe31424b501b13aee6a882ab8a912bee4116